naccs_bot.py

```diff
@@ -40,6 +48,11 @@
 # Mapping of FACEIT Match ID -> List(Discord Voice Channels)
 channels = {}
 
+# Global vars for get_streams()
+DISPLAYEDSTREAMS = {}
```

Comment:
Let's change this to `displayed_streams`. All caps should be reserved for constants